<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="APPLEs-MacBook-Air.local" failures="1" tests="2" name="tests.GoibiboTestRunner" time="14.005" errors="0" timestamp="2021-06-15T12:59:47 IST" skipped="0">
  <testcase classname="tests.GoibiboTestRunner" name="runScenario" time="0.041">
    <failure message="expected [2] but found [1]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [2] but found [1]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:907)
at org.testng.Assert.assertEquals(Assert.java:917)
at tests.TestFeatureSteps.bye(TestFeatureSteps.java:23)
at âœ½.Bye(file:///Users/apple/IdeaProjects/Goibbio_TestNg_Cucumber/src/main/resources/features/Test.feature:5)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase classname="tests.GoibiboTestRunner" name="runScenario" time="13.964"/>
  <system-out/>
</testsuite> <!-- tests.GoibiboTestRunner -->
